app/assets/javascripts/components/calls/IncomingCall.jsx

```diff
@@ -22,7 +22,7 @@ var IncomingCall = React.createClass({
     $('.call-box').toggleClass('hidden');
     var partial = '<div class="notice"><div class="alert alert-danger" role="alert">' +
                     '<span class="glyphicon glyphicon-exclamation-sign" aria-hidden="true"></span>' +
-                      'Missed Call</div></div>';
+                      this.props.helperText.missed + '</div></div>';
```

Comment:
This would typically be associated with something like a constant `MISSED_CALL_MESSAGE`.  Look at a concept called "Replace magic string with symbolic constant."